### DISCUSSION TITLE
Plotting a Confusion Matrix with Creme

### DISCUSSION THREAD

**Discussion by usmanmalik57** [online-ml/river#453] [General]
Hello,
I am working with Creme. I believe that "metric" attribute can only be used to print the running accuracy on the complete dataset while new documents or classes are added. Since we do not have actual and predicted labels for all the previous data points, Is there any way to plot a confusion matrix at a certain point while incremental training?

Regards,
Usman

**Comment by MaxHalford** [comment]
Just to be clear, are you aware that creme and scikit-multiflow have merged into River? If so, have you seen the [confusion matrix documentation](https://riverml.xyz/latest/api/metrics/ConfusionMatrix/)?
